docs/userGuide/addingNavigationButtons.md

```diff
@@ -0,0 +1,190 @@
+{% set title = "Adding Navigation Buttons" %}
+{% set filename = "addingNavigationButtons" %}
+<span id="title" class="d-none">{{ title }}</span>
+
+<frontmatter>
+  title: "User Guide: {{ title }}"
+  layout: userGuide.md
+  pageNav: 2
+</frontmatter>
+
+<span id="link" class="d-none">
+<md>[_User Guide → {{ title }}_]({{ filename }}.html)</md>
+</span>
+
+# {{ title }}
+
+<div class="lead" id="overview">
+
+Navigation buttons can **greatly help your website visitors browse through your content with ease**. 
+</div>
+
+**Overview of steps**
+
+> 1. Define a function to read the page title and create navigation buttons via [Nunjucks](/userGuide/markBindSyntaxOverview.html#support-for-nunjucks)
+> 2. Define a title variable in the content file and invoke the function created to render the navigation buttons
```

Comment:
```suggestion
1. Define a function to read the page title and create navigation buttons via [Nunjucks](/userGuide/markBindSyntaxOverview.html#support-for-nunjucks)
1. Define a title variable in the content file and invoke the function created to render the navigation buttons
```

Reply:
Changed from quotation to points in latest commit!